disco_aws_automation/disco_elb.py

```diff
@@ -57,11 +57,9 @@ def get_certificate_arn(self, dns_name):
 
     def list(self):
         """Returns all of the ELBs for the current environment"""
-        # Grab all of the ELBs in this environment
-        return [
-            elb for elb in
-            throttled_call(self.elb_client.describe_load_balancers).get('LoadBalancerDescriptions', [])
-            if elb['VPCId'] == self.vpc.vpc.id]
+        return [elb for elb in
+                throttled_call(self.elb_client.describe_load_balancers).get('LoadBalancerDescriptions', [])
+                if elb['VPCId'] == self.vpc.get_vpc_id()]
```

Comment:
nitpick, no change required

IMHO pep8 is overly lenient in this indent rule, but I do have to say I prefer indenting 1 level in such cases rather than matching indent level of parenthesis. Why, if you add rewrite statement above you don't need to re-indent everything, but even more importantly 4 space indent is usually less than matching parenthesis -- so you get more on the line without running afoul of 80 char limit (or 120).